### PR TITLE
Update `CHANGELOG.md` for 0.1.0

### PR DESCRIPTION
**Pull request by sunank200**
- Update Changelog for 0.1.0
- Fix the version.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
 # Changelog
 
 Below are the Changelogs for different versions of the universal transfer operator:
```
Comment (phanikumv) [inline docs/CHANGELOG.md:3]:
```suggestion
```

Remove this line please

**Review by phanikumv — APPROVED** [review]

File: src/universal_transfer_operator/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 """An Operator that allows transfers between different datasets."""
 
-__version__ = "0.0.1dev1"
+__version__ = "0.1.0dev1"
```
Comment (utkarsharma2) [inline src/universal_transfer_operator/__init__.py:3]: The version will be 0.1.0, right?

Comment (sunank200) [reply src/universal_transfer_operator/__init__.py:3]: Thats correct.

Comment (utkarsharma2) [reply src/universal_transfer_operator/__init__.py:3]: Ya, okay but now we have to push another commit to release a version update.

**Review by utkarsharma2 — APPROVED** [review]
LGTM

**Pull request merged**